production/helm/loki/templates/_helpers.tpl

```diff
@@ -209,20 +209,26 @@ s3:
 {{- include "loki.lokiStorageConfig" (dict "ctx" . "bucketName" .Values.loki.storage.bucketNames.chunks) | nindent 0 }}
 {{- else if .Values.loki.storage.filesystem }}
 filesystem:
-  {{- toYaml .Values.loki.storage.filesystem | nindent 2 }}
+  {{- with .Values.loki.storage.filesystem }}
+  chunks_directory: {{ .chunks_directory }}
+  rules_directory: {{ .rules_directory }}
+  {{- end }}
```

Comment:
Could you please explain, why you favorite this over https://github.com/grafana/loki/pull/18731?

I coundn't find `admin_api_directory` in GEL and Loki documentation. Removing admin_api_directory feel better then this.

See also https://grafana.slack.com/archives/C09368KL0UR/p1751404079691889